File: src/main/java/com/seth/interviewbit/LengthOfLastWord.java
package com.seth.interviewbit;

public class LengthOfLastWord {

    public static int lengthOfLastWord(final String A) {
        if(A.trim().length() == 0){
            return 0;
        }
        String[] words = A.split(" ");
        String lastWord = words[words.length - 1];
        lastWord = lastWord.trim();
        return lastWord.length();

    }

}

File: src/main/kotlin/com/seth/leetCode/LengthOfLastWord.kt
package com.seth.leetCode

object LengthOfLastWord {
    fun lengthOfLastWord(s: String): Int {
        val string = s.trim()
        if(string.isEmpty()) return 0
        val words = string.split(' ')
        val lastWordIndex = words.size - 1
        val lastWord = words[lastWordIndex]
        return lastWord.length
    }
}